src/Footer/Footer.css

```
@@ -0,0 +1,11 @@
+.app > .footer {
+	background-color: bisque;
+	border-radius: var(--app-padding);
+	/* bottom: 0; */
+	/* margin: 0; */
+	margin-top: var(--app-padding);
+	padding: var(--app-half-padding) 0;
+	/* padding: var(--app-padding); */
+	/* position: absolute; */
+	/* width: calc(100% - 2 * var(--app-padding)); */
+}
```

Always make sure the file has a newline at the end =] the little red circle here in the PR indicates when it is missing

```suggestion
}

```